converter/src/main/java/gov/cms/qpp/conversion/encode/JsonWrapper.java

```diff
@@ -401,7 +401,12 @@ protected int validInteger(String value) {
 	 */
 	protected String validDate(String value) {
 		try {
-			LocalDate thisDate = LocalDate.parse(cleanString(value),  DateTimeFormatter.ofPattern("yyyyMMdd"));
+			String parse = cleanString(value);
+			parse = parse.replace("-", "").replace("/", "");
```

Comment:
How about `parse.replaceAll("[-/]", "");`?

Reply:
I'd rather avoid regex when possible - it's slower, and it's more confusing to read/debug ("Hmm does that character need to be escaped?").

Reply:
Pretty certain it works, tried it against JsonWrapperTest and it passed. Though not clear on whether or not it adversely affects performance

Reply:
I'm not debating that it'll work, just that it's not as practical from a debugging standpoint, and it's likely to be less performant (replaceAll has to compile the pattern every time).

Reply:
k